Description: disable checking if copyright information covers the current year
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: not-needed
Last-Update: 2014-01-05

--- a/test/version.t
+++ b/test/version.t
@@ -34,8 +34,10 @@
 
 my $year = (localtime (time))[5] + 1900;
 
+SKIP: { skip 'fails in year 2014 and later', 1;
 my $output = qx{../src/task rc:version.rc version 2>&1};
 like ($output, qr/Copyright \(C\) \d{4} - $year/, 'Copyright is current');
+}
 
 # Cleanup.
 unlink 'version.rc';
